recurly/__init__.py

```diff
@@ -1061,8 +1080,11 @@ def filter_currency(resources):
         except AttributeError:
             pass
 
+        return self.collect(url, self)
+
+    def collect(self, url, body=None):
```

Comment:
The name `collect` could lead to confusion as it's is fairly synonymous with `capture`. It should also start with double underscores to suggest that it is a private method and not for external consumption.

Reply:
Naming things is hard.